website/docs/overview.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
+
+
+
+## 0.1 About the Syllabus
+This syllabus serves as both a guidance document for participants and a requirement specification for Accredited Training Providers
+preparing candidates for the "Robot Framework® Certified Professional" (RFCP®) exam.
+It outlines the structure, learning objectives, and knowledge areas essential for certification.
+
+This syllabus is not a training manual, tutorial, or comprehensive learning resource but instead defines the scope of knowledge
+that must be taught in a Robot Framework training and acquired by participants to meet the certification requirements.
+
+:::note
+
+For further explanation, complementary examples, and practical exercises beyond the scope of this syllabus,
+participants are encouraged to consult additional resources, such as the official documentation at [docs.robotframework.org](https://docs.robotframework.org/).
+
+:::
+
+The syllabus is divided into chapters that progress logically from basic concepts to more advanced topics of Robot Framework.
+
+The learning objectives (LOs) specified within this document are binding,
+meaning they define the specific knowledge and skills participants are expected to acquire during the course in order to pass the exam.
+Therefore, trainers are required to effectively cover the syllabus within their course.
+Additionally, the recommended sequence of topics in this syllabus helps guide the order of learning,
+but the specific teaching methods, order and pace may be adapted by the instructor based on class dynamics or need.
+
+
+
+
+## 0.2 About "Robot Framework® Certified Professional"
```

Comment:
Should the syllabus also explain the exam a bit? As in how many questions it will have, what types of questions, weight on different question domains, etc.?